src/views/MainPage.vue

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable vue/no-parsing-error */
+<template>
+  <v-container>
+      <div v-for="film in films" :key="film.id" class="card-film">
+      <div class="card-film__img">
+        <img :src="'https://image.tmdb.org/t/p/w500' + film.poster_path" alt="poster">
+      </div>
+      <div class="card-film__descr">
+        <div class="card-film__descr__name">
+          Name of film: {{ film.original_title }}
+        </div>
+        <div class="card-film__descr__overview">
+          Overview: {{ film.overview }}
```

Comment:
try to wrap Overview: and interpolation in separate elements:
```
<div>
  <span>Overview:</span>
  <span>{{ film.overview }}</span>
</div>
```

Reply:
Done